wdl/tasks/picard.wdl

```diff
@@ -65,3 +65,34 @@ task mark_duplicates_with_mate_cigar {
         disks: resources.disks
     }
 }
+
+
+task collect_insert_size_metrics {
+    input {
+        File bam
+        CollectInsertSizeMetricsParams params
+        Resources resources
+        String out = "CollectInsertSizeMetrics.picard"
+        String pdf_out = "CollectInsertSizeMetrics.picard.pdf"
+    }
+
+    command {
+        java -jar $(which picard.jar) CollectInsertSizeMetrics \
+            INPUT=~{bam} \
```

Comment:
Wrap these like https://github.com/ENCODE-DCC/dnase-seq-pipeline/blob/7de1358428d85070b9b641265d07e06a06b537fd/wdl/tasks/picard.wdl#L48-L49

Reply:
I on purpose did not. The ones unwrapped are mandatory ones.

Reply:
Should make it consistent with what's already in the file.